src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -1842,7 +1840,15 @@ SystemNative_SetSockOpt(intptr_t socket, int32_t socketOptionLevel, int32_t sock
                 }
             }
 
+            // An application that sets SO_REUSEPORT/SO_REUSEADDR can share the with another application
```

Comment:
share the .. ?

Reply:
"endpoint", pushed a commit to fix this.